APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1141,7 +1208,36 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 }
             }
                 break;
-            
+                
+            case kAPCUserInfoItemTypeDateOfBirth:
+            {
+                if (self.canEditBirthDate)
```

Comment:
Erin, I updated this conditional to hopefully be more readable and added some comments.  In short, if you enabled "canEditBirthdate" it will run through, now a totally separate conditional branch, than the APCDefaultTableViewCell, since it needs to check for the conditions below.